File: src/main/java/pr/eleks/we_at_her/services/data/CityService.java
package pr.eleks.we_at_her.services.data;

import pr.eleks.we_at_her.dto.CityDto;
import pr.eleks.we_at_her.entities.City;

import java.util.List;

public interface CityService {
    List<CityDto> findAllCities();

    CityDto convertToDto(City city);

    City convertToEntity(CityDto cityDto);
}

File: src/main/kotlin/pr/eleks/we_at_her/services/data/CityService.kt
package pr.eleks.we_at_her.services.data

import pr.eleks.we_at_her.dto.CityDto
import pr.eleks.we_at_her.entities.City

interface CityService {
    fun findAllCities(): MutableList<CityDto>
    fun convertToDto(city: City): CityDto
    fun convertToEntity(cityDto: CityDto): City
}